config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -65,6 +65,25 @@ spec:
                     0 in case of no traffic
                   format: int64
                   type: integer
+                pytorch:
+                  properties:
+                    modelClassFile:
+                      type: string
+                    modelClassName:
+                      description: Defaults to latest PyTorch Version.
```

Comment:
I think you need to move the comment on the go code...

Reply:
Done